_includes/installation.html

```diff
@@ -86,7 +86,7 @@ <h3>Usage Example</h3>
 	<!-- End Empty Results -->
 
 	<div class="latest python">
-		{% highlight bash %}pip install duckdb=={{ site.currentduckdbversion }}{% endhighlight %}
+		{% highlight bash %}pip install duckdb{% endhighlight %}
```

Comment:
If we're going to remove the explicit version, may be worth adding the -U flag

Reply:
Sure, see #2216